### PR TITLE
Ensure right-click on run mode does not unhilite a radio button

### PR DESCRIPTION
**Pull request by livecodepanos**

### CODE REVIEW

File: engine/src/group.cpp [hunk 1]
```diff
@@ -891,7 +891,8 @@ Boolean MCGroup::mup(uint2 which, bool p_release)
 	newkfocused = mfocused;
 	MCControl *oldfocused = mfocused;
 	// MH-2007-03-20: [[ Bug 705 ]] Selecting a radio button using pointer tool unhilites other radio buttons in the group with radiobehavior set.
-	if (tool != T_POINTER)
+    // PM-2018-09-21: [[ Bug 9711 ]] Ensure right-clicking on a grouped radio button does not change the hilite of the group
+	if (tool != T_POINTER && which == 1)
```
Comment (montegoulding) [inline engine/src/group.cpp]: @livecodepanos you should use `Button1` here I think instead of `1`

**Comment by montegoulding** [comment]
@livecode-vulcan review ok 3362058

**Comment by livecode-vulcan** [comment]
💙 review by @montegoulding ok 3362058e3392e2a218d391f89f9560ce2e08d373

**Comment by livecode-vulcan** [comment]
😞 test failure 3362058e3392e2a218d391f89f9560ce2e08d373

* try-community-armv6-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-sdk26_ndk16r15/builds/103)
* try-community-armv7-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv7-android-sdk26_ndk16r15/builds/95)
* try-community-arm64-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-arm64-android-sdk26_ndk16r15/builds/95)
* try-community-x86-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86-android-sdk26_ndk16r15/builds/95)
* try-community-x86_64-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86_64-android-sdk26_ndk16r15/builds/95)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/1617)
* try-community-universal-ios-iphoneos11.4: [failure](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos11.4/builds/144)
* try-community-universal-ios-iphonesimulator11.4: [failure](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator11.4/builds/144)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/1647)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/1640)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/1637)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/1673)
* try-community-x86_64-win32: [pending](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/1614)

**Comment by livecodepanos** [comment]
@livecode-vulcan retry 3362058

**Comment by livecode-vulcan** [comment]
😎 test success 3362058e3392e2a218d391f89f9560ce2e08d373

* try-community-armv6-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-sdk26_ndk16r15/builds/105)
* try-community-armv7-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv7-android-sdk26_ndk16r15/builds/97)
* try-community-arm64-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-arm64-android-sdk26_ndk16r15/builds/97)
* try-community-x86-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86-android-sdk26_ndk16r15/builds/97)
* try-community-x86_64-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86_64-android-sdk26_ndk16r15/builds/97)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/1619)
* try-community-universal-ios-iphoneos12.0: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos12.0/builds/1)
* try-community-universal-ios-iphonesimulator12.0: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator12.0/builds/1)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/1649)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/1642)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/1639)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/1675)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/1616)

**Pull request merged**